src/app.config.js

```diff
@@ -1,5 +1,5 @@
 // Copyright (c) Microsoft. All rights reserved.
-const baseUrl = process.env.REACT_APP_BASE_SERVICE_URL || '';
+const baseUrl = process.env.REACT_APP_BASE_SERVICE_URL || 'https://maryellen-0226.azurewebsites.net';
```

Comment:
Remove: 'https://maryellen-0226.azurewebsites.net' Does the .env value doesn't work?

Reply:
woops